src/auth/components/EmailVerificationPage/index.tsx

```diff
@@ -1,8 +1,8 @@
 import React, { useEffect, useState } from 'react';
 import { Button } from 'antd';
 import { useParams, useHistory } from 'react-router-dom';
-import { verifyEmail } from '../../authActions';
 import './style.less';
+import { verifyEmail } from '../../utils';
```

Comment:
Can we keep `import './style.less';` as the last import?

Reply:
For every file that I modified, I ran Typescript's built in `organizeImports` command from the language server `tsserver`. I'm not sure if we can configure it, but I like having an automated, opinionated formatter for imports.